Makefile

```diff
@@ -47,6 +65,10 @@ linux-prepare-$(LINUX_DISTRO)::
 linux-prepare-$(LINUX_DISTRO)-$(LINUX_DISTRO_RELEASE)::
 endif
 
+# This must come last! It needs packages installed above
+prepare::
+	./build-tools/scripts/generate-os-info Configuration.OperatingSystem.props
```

Comment:
This is oddly duplicative of `make prepare-props`, above. Should we delete the `generate-os-info` invocation from `make generate-props` and just rely on this one? Or should it be a `prepare-props::` double-colon target?